sonic_platform_base/sonic_xcvr/api/public/c_cmis.py

```diff
@@ -0,0 +1,110 @@
+"""
+    c-cmis.py
+
+    Implementation of XcvrApi that corresponds to C-CMIS
+"""
+from ...fields import consts
+from ..xcvr_api import XcvrApi
+import time
+BYTELENGTH = 8
+class CCmisApi(XcvrApi):
```

Comment:
This shouldn't inherit from XcvrApi. CCmisApi would only contain API specific to C-CMIS and would not implement XcvrApi methods. It can just inherit from object instead.